packages/thirdweb/src/wallets/in-app/native/helpers/analytics.ts

```diff
@@ -0,0 +1,5 @@
+import { randomBytes } from "../../../../utils/random.js";
+
+export const ANALYTICS = {
+  nonce: randomBytes(16),
```

Comment:
wat

Reply:
Yeah it's really weird. Though I think it's actually used and important. @elasticbottle can you confirm?

Reply:
this is actually used, but for tracking purposes only. all requests seems to be logged with this nonce header

not sure if we actually use to track stuff, maybe to get a view of an entire login session like latency?

Reply:
I guess that's fine but I would've expected us to move it to where we do any fetches (and just call a function then that returns the "cached" value if its already been called) -> right now this will generate random bytes the moment this file gets included in the bundle which seems like the wrong time to do it

Reply:
yeah ill change that